src/Aspire.ProjectTemplates/cgmanifest.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+    "version": 1,
+    "registrations": [
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Exporter.OpenTelemetryProtocol",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Extensions.Hosting",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.AspNetCore",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.Http",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.Runtime",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "Microsoft.Extensions.Http.Resilience",
```

Comment:
why this one specifically when templates have these versioned the same way?
```
<PackageReference Include="Aspire.Hosting.AppHost" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
<PackageReference Include="Aspire.Hosting.Redis" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
<PackageReference Include="Aspire.Hosting.Testing" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
<PackageReference Include="Aspire.StackExchange.Redis.OutputCaching" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
<PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="!!REPLACE_WITH_EXTENSIONS_VERSION!!" />
<PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
```

Reply:
oh I guess you just haven't pulled it yet per above

Reply:
`Microsoft.Extensions.Http.Resilience` is using `REPLACE_WITH_EXTENSIONS_VERSION`, not `REPLACE_WITH_LATEST_VERSION` which represents locally built packages that CG would pick as a standard artifact.

Reply:
Ah - thanks - my eyeballs missed that.